lib/admin.php

```diff
@@ -209,7 +226,11 @@ function wc_admin_admin_after_notices() {
 }
 add_action( 'admin_notices', 'wc_admin_admin_after_notices', PHP_INT_MAX );
 
-// TODO Can we do some URL rewriting so we can figure out which page they are on server side?
+/**
+ * TODO Can we do some URL rewriting so we can figure out which page they are on server side?
```

Comment:
Better include a properly docblock here, and then use `@todo`, we use this on core and will make easy to find TODOs later.